odoo-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Fail on any error
+#
+set -e
+
+# Construct Odoo invocation command
+#
+command="/opt/odoo/odoo-bin --database odoo --stop-after-init"
+
+# Extend Odoo command to get coverage tests for any (non-core) addons
+#
+addonsdir=/opt/odoo-addons
+addonspath=/opt/odoo/odoo/addons,/opt/odoo/addons
+addons=$(find ${addonsdir} -mindepth 1 -maxdepth 1 -type d -printf '%f\n')
+if [ -n "${addons}" ] ; then
+    for addon in ${addons} ; do
+	if [ -n "${includes}" ] ; then
+	    includes="${includes},"
+	fi
+	includes="${includes}*addons/${addon}/*"
+    done
+    command="coverage3 run --include=\"${includes}\" --omit=\"*/tests/*\" \
+             --branch ${command}"
+    addonspath="${addonspath},${addonsdir}"
+fi
+command="${command} --addons-path ${addonspath}"
+
+# Append any script arguments to the Odoo command
+#
+if [ -n "$*" ] ; then
+    command="${command} $*"
+fi
+
+# Echo invocation commands for debugging
+#
```

Comment:
Q: Why all the empty # lines?

Reply:
Some parts are copy pasted from https://github.com/unipartdigital/odoo-tester/blob/14.0/odoo-wrapper as that's been bundled in with this image - i've left the formatting quirks in